src/sources/util/http.rs

```diff
@@ -74,15 +76,20 @@ pub trait HttpSource: Clone + Send + Sync + 'static {
             .and(warp::path::end())
             .and(warp::header::headers_cloned())
             .and(warp::body::bytes())
-            .and_then(move |headers: HeaderMap, body| {
+            .and_then(move |headers: HeaderMap, body: Bytes| {
```

Comment:
Is this required change?

Reply:
yes, rustc couldn't infer the type when I add `let body_size = body.len();`.